src/ros/rviz_grid_viewer.h

```diff
@@ -21,8 +21,9 @@ class RvizGridViewer {
  * Initializes a map and robot's position publisher.
  * \param pub A map publisher to ROS.
  */
-  RvizGridViewer(ros::Publisher pub, const double show_map_rate) :
-    _map_pub(pub), map_publishing_rate(show_map_rate) {}
+  RvizGridViewer(ros::Publisher pub, const double show_map_rate, 
+                 std::string odom, std::string robot_pose) :
```

Comment:
"odom" name is not meaningful (string doesn't hold raw odometry) as well as robot_pose